app/src/main/java/io/apicurio/registry/storage/impl/sql/RegistryDatasourceProducer.java

```diff
@@ -49,6 +49,10 @@ public class RegistryDatasourceProducer {
     @Info(category = "storage", description = "Application datasource pool maximum size", availableSince = "3.0.0")
     String maxSize;
 
+    @ConfigProperty(name = "apicurio.datasource.jdbc.max-life-time", defaultValue = "0")
```

Comment:
Please add a `.seconds` suffix to the property name, i.e. `apicurio.datasource.jdbc.max-life-time.seconds`.